OpenSim/Analyses/JointReaction.cpp

```diff
@@ -297,24 +295,16 @@ void JointReaction::setupReactionList()
             // determine if user wants reaction on child or parent
             currentKey.isAppliedOnChild = (appliedOnName == "child");
             currentKey.appliedOnBody = currentKey.isAppliedOnChild ?
-                &joint.getChildFrame() : &joint.getParentFrame();
+                &joint.getChildFrame().findBaseFrame() : &joint.getParentFrame().findBaseFrame();
 
+            // set frame that reactions are expressed in
             std::string expressedIn = "ground";
             if (_inFrame.size()) {
                 expressedIn = (i < _inFrame.size()) ? _inFrame[i] : _inFrame[0];
             }
-            // convert to lowercase
-            std::transform(expressedIn.begin(), expressedIn.end(),
-                expressedIn.begin(), ::tolower);
-            if (expressedIn == "child"){
-                currentKey.expressedInFrame = &joint.getChildFrame();
-            }
-            else if (expressedIn == "parent"){
-                currentKey.expressedInFrame = &joint.getParentFrame();
-            }
-            else{ //if not child or parent use ground
-                currentKey.expressedInFrame = &_model->getGround();
-            }
+            _model->dumpSubcomponents();
```

Comment:
Did you intend to leave this call to `dumpSubcomponents()` here? It seems like this would display more information than would be useful, particularly if the model is large.